### PR TITLE
Add real time replication metrics 

### PR DESCRIPTION
**Pull request by ssepml**
## Description

Add missing real time replication metrics.

Also, keep the full name for fullsync metrics.

### CODE REVIEW

**Review by kivra-joarod — APPROVED** [review]
For me looks good. But I am not the best person to review this.

File: src/riak_exporter/exporter.py [hunk 1]
```diff
@@ -89,9 +89,25 @@ def parse_riak_repl_stats_data(self, riak_repl_stats_data):
...
+                        prom_str = "riak_repl_realtime_queue_stats_{} {}".format(rkey, prom_value)
+                        yield prom_str
+
+                    if rkey == "consumers":
+                        for cluster in rvalue:
+                            for ckey, cvalue in value[cluster].items():
+                                if isinstance(cvalue, (int, float, bool)):
+                                    prom_value = float(cvalue)
+                                    prom_str = "riak_repl_realtime_queue_stats_consumers_{}{{cluster=\"{}\"}} {}".format(rkey, cluster, prom_value)
+                                    yield prom_str
```
Comment (RSBRK) [inline src/riak_exporter/exporter.py:108]: Not asking for it now, but this sequence seems to be repeated many times, maybe we can make a function of it.

**Review by RSBRK — APPROVED** [review]

File: src/riak_exporter/exporter.py [hunk 2]
```diff
@@ -89,9 +89,25 @@ def parse_riak_repl_stats_data(self, riak_repl_stats_data):
                     for ckey, cvalue in value[cluster].items():
                         if isinstance(cvalue, (int, float, bool)):
                             prom_value = float(cvalue)
-                            prom_str = "riak_repl_{}{{cluster=\"{}\"}} {}".format(ckey, cluster, prom_value)
+                            prom_str = "riak_repl_fullsync_coordinator_{}{{cluster=\"{}\"}} {}".format(ckey, cluster, prom_value)
                             yield prom_str
 
+            if key == "realtime_queue_stats":
+                for rkey, rvalue in value.items():
```
Comment (RSBRK) [inline src/riak_exporter/exporter.py:96]: Just curious, what does the r in `rkey` and `rvalue` mean? The c in `cvalue` and `ckey` I assume is for `cluster`. But then the r gets a bit confusing if it's `right-hand-value` or `realtime_queue_stats`-`value`

File: src/riak_exporter/exporter.py (hunk 1 above)
Comment (ssepml) [reply src/riak_exporter/exporter.py:108]: It's not that easy due to the string and the number of arguments, so I didn't spend time on it

File: src/riak_exporter/exporter.py (hunk 2 above)
Comment (ssepml) [reply src/riak_exporter/exporter.py:96]: Comes from `real_time` and to differentiate from the upper level for loop

**Pull request merged**